packages/guesstimator/formatter/formatters/lib.js

```diff
@@ -74,8 +76,9 @@ export function shorthandIntoLognormalFormattingStep(text){
       
         let low = arrayLowHigh[0]
         let high= arrayLowHigh[1]
-
-        return distributionUpToIntoLognormal(low, high)
+        
+        result = (getGuesstimateType("unknown yet", [low])=='LOGNORMAL') ? distributionUpToIntoLognormal(low, high) : distributionUpToIntoNormal(low, high)
```

Comment:
I imagine there must be some cleaner way of doing this

Reply:
Yes, but not if you want to reuse code.